### PR TITLE
Create a version 7.0 of web_action_conditionable module

### PR DESCRIPTION
**Pull request by kossovo**

### CODE REVIEW

File: web_action_conditionable/README.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+* ...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/web/issues/new?body=module:web_action_conditionableversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
```
Comment (petrus-v) [inline web_action_conditionable/README.rst:51]: The link should be https://github.com/OCA/web/issues/new?body=module:web_action_conditionableversion:%207.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**

(version: 7.0 instead 8.0)

**Comment by petrus-v** [comment]
Great, thanks for this backport :+1:

File: web_action_conditionable/README.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```
Comment (elicoidal) [inline web_action_conditionable/README.rst:4]: Hi @kossovo thanks for the contribution
Please follow the newest OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
You should have a document title (add ======================== before)

"This module extends the functionality of actions in tree views."
s/supports/supports

File: web_action_conditionable/README.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+To use this module, you need to:
+
+* go to ...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/web/7.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (elicoidal) [inline web_action_conditionable/README.rst]: remove those 2 lines

File: web_action_conditionable/README.rst [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+.. branch is "7.0" for example
+
+For further information, please visit:
```
Comment (elicoidal) [inline web_action_conditionable/README.rst]: remove forum section

File: web_action_conditionable/README.rst [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```
Comment (elicoidal) [inline web_action_conditionable/README.rst]: if empty remove

File: web_action_conditionable/README.rst [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline web_action_conditionable/README.rst]: https

File: web_action_conditionable/__init__.py [hunk 7]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline web_action_conditionable/__init__.py:1]: Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

**Comment by kossovo** [comment]
@petrus-v @elicoidal Thanks for the reviews. 
I made some changes and I think that everything is ok now.

File: web_action_conditionable/README.rst [hunk 8]
```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=========================
+Web Action Conditionable
+=========================
+
+This module was written to extend the functionality of actions in tree views.
```
Comment (elicoidal) [inline web_action_conditionable/README.rst:9]: This module extends

File: web_action_conditionable/README.rst [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+=============
+
+Not needed.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to ...
```
Comment (elicoidal) [inline web_action_conditionable/README.rst]: remove line

File: web_action_conditionable/__init__.py [hunk 10]
```diff
@@ -0,0 +1,3 @@
...
+# © <2015> <Cristian Salamea, Clovis Nzouendjou, Odoo Community Association (OCA)>
```
Comment (elicoidal) [inline web_action_conditionable/__init__.py]: no need the <>

File: web_action_conditionable/__openerp__.py [hunk 11]
```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# © <2015> <Cristian Salamea, Clovis Nzouendjou, Odoo Community Association (OCA)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": 'web_action_conditionable',
+    "version": "7.0.0.1.0",
```
Comment (elicoidal) [inline web_action_conditionable/__openerp__.py]: 7.0.1.0.0

**Comment by elicoidal** [comment]
some small details. Other than that LGTM (no technical review)

**Pull request closed without merging**